### PR TITLE
[WX-1361] Remove Confusing Error Message

### PR DESCRIPTION
**Pull request by THWiseman**
### Description

When Cromwell restarts during a failure, it must fail the 'next' upcoming tasks in order to cleanly terminate the workflow. During this process, it was logging an error that isn't really an error (and was very confusing to users). 

This PR: 
- Changes the failure reason to something more relevant.
- Removes the failure reason from the 'Workflow' level failures, since it was not the reason the workflow failed. The failure reason is still attached to the task that never ran. 

#### `CHANGELOG.md`
 - [x] I updated `CHANGELOG.md` in this PR
 - [ ] I assert that this change shouldn't be included in `CHANGELOG.md` because it doesn't impact community users

#### Terra Release Notes
 - [x] I added a suggested release notes entry in this Jira ticket
 - [ ] I assert that this change doesn't need Jira release notes because it doesn't impact Terra users

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## 88 Release Notes
 
+### Improved logging behavior
+When Cromwell restarts during a workflow that is failing, it no longer reports pending tasks as a reason for that failure.
```
Comment (aednichols) [inline CHANGELOG.md:6]:
```suggestion
### Improved status reporting behavior

When Cromwell restarts during a workflow that is failing, it no longer reports pending tasks as a reason for that failure. 
```

**Review by aednichols — APPROVED** [review]
Nice work finding a concise solution!

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala [hunk 2]
```diff
@@ -470,11 +477,14 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
                                         failedJobKey: JobKey,
                                         reason: Throwable,
                                         returnCode: Option[Int] = None,
-                                        jobExecutionMap: JobExecutionMap = Map.empty
+                                        jobExecutionMap: JobExecutionMap = Map.empty,
+                                        omitFromWorkflowLevelFailures: Boolean = false
```
Comment (jgainerdewar) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala:481]: Nitpick that you can feel free to ignore if you disagree - IMO boolean arguments where `true` represents negation require readers to think too hard. I think it's easier to understand if it's `includeInWorkflowLevelFailures: Boolean = true`.

**Review by jgainerdewar — APPROVED** [review]
Nice job finding such a small and tidy solution!

**Pull request merged**